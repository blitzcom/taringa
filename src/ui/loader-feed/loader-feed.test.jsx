import renderer from 'react-test-renderer';

import LoaderFeed from './loader-feed';

describe('ui/loader-feed', () => {
  def('items', () => [1, 2]);
  def('subject', () => renderer.create(<LoaderFeed items={$items} />).toJSON());

  it('renders loader', () => {
    expect($subject).toMatchSnapshot();
  });
});
