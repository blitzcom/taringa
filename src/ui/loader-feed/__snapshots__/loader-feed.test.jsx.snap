// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/loader-feed renders loader 1`] = `
<div
  className="loader"
>
  <div
    className="wrapper"
  >
    <div
      className="meta"
    >
      <div
        className="title"
      />
      <div
        className="subtitle"
      />
    </div>
    <div
      className="thumbnail"
    />
  </div>
  <div
    className="wrapper"
  >
    <div
      className="meta"
    >
      <div
        className="title"
      />
      <div
        className="subtitle"
      />
    </div>
    <div
      className="thumbnail"
    />
  </div>
</div>
`;
